frontend/src/app/components/edit-sprint/edit-sprint-dialog/edit-sprint-dialog.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { Subject } from 'rxjs';
+import { EditSprintModel } from 'src/core/models/sprint/edit-sprint-model';
+import { TasqueDropdownOption } from 'src/shared/components/tasque-dropdown/dropdown.component';
+import * as moment from 'moment'; 
+import { SprintService } from 'src/core/services/sprint.service';
+import { NotificationService } from 'src/core/services/notification.service';
+
+@Component({
+  selector: 'app-edit-sprint-dialog',
+  templateUrl: './edit-sprint-dialog.component.html',
+  styleUrls: ['./edit-sprint-dialog.component.sass']
+})
+export class EditSprintDialogComponent implements OnInit {
+
+  public createBtnClass = 'fill';
+  public cancelBtnName = 'Cancel';
+  public cancelBtnClass = 'fill gray';
+  public inputClass = 'input';
+  public requiredErrorMessage = 'This field is required.';
+
+  public sprintForm: FormGroup = new FormGroup({});
+  public sprintName: FormControl;
+  public sprintDuration: FormControl;
+  public sprintStartAt: FormControl;
+  public sprintEndAt: FormControl;
+  public sprintDescription: FormControl;
+
+  public duration: TasqueDropdownOption;
+
+  public periods: TasqueDropdownOption[] = [
+    { color: '#0000', title: 'Custom', id: 0 },
```

Comment:
If you don't need any color, you may not add it

Reply:
And change in [tasque-dropdown.ts](https://github.com/BinaryStudioAcademy/bsa-2022-tasque/blob/dev/frontend/src/shared/components/tasque-dropdown/dropdown.component.ts), please, from it
```ts
export interface TasqueDropdownOption {
  color: string;
  title: string;
  id: number;
}
```
To
```ts
export interface TasqueDropdownOption {
  color?: string;
  title: string;
  id: number;
}
```
Because somebody overwrote my changes